app-misc/elasticsearch/elasticsearch-5.6.6.ebuild

```diff
@@ -20,21 +20,21 @@ pkg_setup() {
 }
 
 src_prepare() {
-	rm -v bin/*.{bat,exe} LICENSE.txt || die
-
 	default
+
+	rm -v bin/*.{bat,exe} LICENSE.txt || die
 }
 
 src_install() {
 	keepdir /etc/${PN}
```

Comment:
Why do you need to keepdir it if you install files there? I don't see how it would wind up empty.